styles/twitter/catppuccin.user.css

```diff
@@ -397,6 +417,42 @@
       color: @overlay0;
     }
 
+    // media
+    // .r-loe9s5 {
+    //  background-color: fade(@crust, 77%)
+    // }
+
```

Comment:
```suggestion
```
Don't need this right?

Reply:
pppprobably. i just put it there so i can look back on it whenever i get the time